util/gh-pages/index.html

```diff
@@ -89,7 +89,7 @@ <h1>ALL the Clippy Lints</h1>
             </div>
 
             <article class="panel panel-default" id="{{lint.id}}"
-                ng-repeat="lint in data | filter:byLevels | filter:byGroups | filter:search | orderBy:'id' track by lint.id" on-finish-render="ngRepeatFinished">
```

Comment:
I removed the `on-finish-render="ngRepeatFinished"` as it was restarting the query even though nothing has changed. Changing one of the settings causes the query to me executed automatcially